app/controllers/candidate_interface/application_form_controller.rb

```diff
@@ -1,12 +1,14 @@
 module CandidateInterface
   class ApplicationFormController < CandidateInterfaceController
+    before_action :redirect_to_dashboard_if_submitted, only: :review
```

Comment:
```suggestion
    before_action :redirect_to_dashboard_if_submitted, only: %i[review submit_show submit submit_success]
```

Should we redirect these pages too?

Reply:
Infinite redirects unless we refactor the controller actions :)